examples/oft-solana/tasks/solana/sendOFT.ts

```diff
@@ -72,6 +72,15 @@ task('lz:oft:solana:send', 'Send tokens from Solana to a target EVM chain')
                 )
             }
 
+            const tokenAccountData = await fetchToken(umi, tokenAccount)
+            const balance = Number(tokenAccountData.amount)
```

Comment:
This is not on you, but is there any chance you can actually make this bigint?  This easily overflows.  Also will need to change amount on line 25 to `types.bigint`.  I apologize, this is my fault.

Reply:
ahh, that's a good catch. all good! doing it now.

Reply:
pushed in latest commit

Reply:
@ryandgoulding lmk if other changes need to be made. Can't merge this PR without resolving your change request.